plugins/cj-switch.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: cj-switch
+spec:
+  description: |
+    The list of the features:
+    * Invert the spec.suspend flag. 'True' turns to 'False' and vice versa
+    * Save current state to and restore state from file
+    * Overwrite spec.suspend flag
+    Possible filters:
+      * All namespaces
+      * Filter by label
+      * Filter by namespace
+    Example:
+      #
+      # Inverts spec.suspend flag for all namespaces or specific one.
+      #
+      kubectl cj-switch [-A|--all-namespaces|-n|--namespace]
+      #
+      # Inverts spec.suspend flag of all cron jobs with the specific label. Could be used in conjunction with namespace options, -A -n etc.
+      #
+      kubectl cj-switch -l key=value
+      #
+      # Sets spec.suspend flag to false/true for all cron jobs within default namespace.
+      #
+      kubectl cj-switch -n default -f ./<FILENAME> -s [false|true]
+      #
+      # Restore state from the file
+      #
+      kubectl cj-switch -f ./<FILENAME> -r
+  homepage: https://github.com/pymag09/kubectl_cj-switch
+  platforms:
+  - bin: "./cj-switch.sh"
+    files:
+    - from: /*
+      to: .
```

Comment:
This is the default. No need to spell it out.

Reply:
Unfortunately without this i see the following error:
> Selector:nil Files:[{From: To:}] Bin:./cj-toggle.sh}) is badly constructed: `files` is invalid: `from` field has to be

Reply:
🤔 did you drop only one of `from` and `to`? This error is peculiar.

Reply:
I dropped both but I left `files:` Probably this caused the error.